pkg/scheduler/api/devices/util.go

```diff
@@ -0,0 +1,38 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package devices
+
+// These are predefined codes used in a Status.
+const (
```

Comment:
These same status are defined in device package and api package. Is it possible to combine two to one?

Reply:
The Volcano API package references the devices package. If the status definition in the API is used, cyclic dependency occurs.